docs/docs/README.md

```diff
@@ -19,9 +19,26 @@ This project can be used for bootstrapping Atlassian Bamboo DC in a K8s cluster.
 
 ## Deployment overview
 
-The diagram below provides a high level overview of what a typical deployment will look like:
+The diagram below provides a high level overview of what a typical deployment will look like for each DC product:
+
```

Comment:
The product order in variable and config file is Jira, Confluence, and Bamboo. So is good to follow this order in documentation as well.

Reply:
Not sure I see a need for this following the ordering of whats in `tfvars.config`. From a documentation standpoint I think alphabetic makes more sense.